web/src/CityContentSwitcher.tsx

```diff
@@ -36,17 +36,18 @@ import {
   TU_NEWS_DETAIL_ROUTE,
   TU_NEWS_ROUTE
 } from './routes'
-
-const TuNewsDetailPage = React.lazy(() => import('./routes/TuNewsDetailPage'))
-const TuNewsPage = React.lazy(() => import('./routes/TuNewsPage'))
-const OffersPage = React.lazy(() => import('./routes/OffersPage'))
-const EventsPage = React.lazy(() => import('./routes/EventsPage'))
-const CategoriesPage = React.lazy(() => import('./routes/CategoriesPage'))
-const LocalNewsPage = React.lazy(() => import('./routes/LocalNewsPage'))
-const SprungbrettOfferPage = React.lazy(() => import('./routes/SprungbrettOfferPage'))
-const PoisPage = React.lazy(() => import('./routes/PoisPage'))
-const SearchPage = React.lazy(() => import('./routes/SearchPage'))
-const DisclaimerPage = React.lazy(() => import('./routes/DisclaimerPage'))
+import retryImport from './utils/retryImport'
+
+const TuNewsDetailPage = React.lazy(() => retryImport(() => import('./routes/TuNewsDetailPage')))
```

Comment:
Could you perhaps refactor your `retryImport` to something like `lazyWithRetry`? I.e. move the `React.lazy` inside it such that you can write the following:
```suggestion
const TuNewsDetailPage = lazyWithRetry('./routes/TuNewsDetailPage')
```
or
```suggestion
const TuNewsDetailPage = lazyWithRetry(() => import('./routes/TuNewsDetailPage'))
```

Reply:
good advice. done